sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,4 @@
+SELECT "filmId",
+       COUNT(*) AS "totalInv"
+  FROM "inventory"
+  GROUP BY "filmId";
```

Comment:
This should be a single count of all the film copies in inventory, not a count for each film.

---

![image](https://user-images.githubusercontent.com/16143771/131412176-ba9c6290-d091-4c02-9249-d543e2dc30de.png)